js/queen.js

```diff
@@ -52,7 +57,11 @@ export class Queen extends Piece{
          * Get row decreasing moves
          */
         for (let i = this.rowPos - 1; i >= minRows; i--) {
-            if (board[i][this.colPos] !== undefined && board[i][this.colPos].getPlayerId() == this.playerId) {
+            if (board[i][this.colPos] && board[i][this.colPos].getPlayerId() == this.playerId) {
```

Comment:
`===` should be used instead of `==` on line `60`.

Reply:
Updated.